src/deploy/NVA_build/rsyslog.conf

```diff
@@ -49,6 +49,21 @@ $OmitLocalLogging off
 # Logging much else clutters up the screen.
 #kern.*                                                 /dev/console
 
+if $msg contains '{"op":"PUT"' then {
```

Comment:
I suggest selecting messages based on a single tag that will be added to all bucket logging messages, instead of selecting only specific ops

Reply:
Makes sense. will modify it